src/utils.js

```diff
@@ -0,0 +1,10 @@
+const formatDate = (date) => {
+  const dateTime = new Date(date);
+  const year = dateTime.toLocaleDateString("en-US", { year: "numeric" });
+  const month = dateTime.toLocaleDateString("en-US", { month: "short" });
+  const day = dateTime.toLocaleDateString("en-US", { day: "numeric" });
+
+  return `${day} ${month} ${year}`;
+};
+
+export default formatDate;
```

Comment:
use export const instead. `utils.js` can have multiple functions in the future.

Reply:
Done.